Cookbook/Technical-Documents/ReleaseProcess.md

```diff
@@ -25,101 +25,151 @@ The bug's corresponding feature should be disabled altogether using its feature
 
-    1. Send the build to TestFlight Beta (external testing). Select the `External Testers` group.
-    1. Press `Release this version` in App Store Connect
     1. Create a tag named `{appname}/{version}` (e.g. `babylon/4.1.0`) on the release commit and create a GitHub release for that new tag
        * Make sure you create separate tags (and GitHub releases) for each app released on the AppStore (eg. Babylon 4.1.0 and Telus 4.1.0 would each have their own `babylon/4.1.0` and `telus/4.1.0` tags)
        * Set the body of the GitHub release to the content of the Release Notes for the app
        * Attach the zipped `xcarchive` as an artefact to the GitHub release (if you're using the automated release command, you can find the `*.xcarchive.zip` in the Artifacts top section in the CI build).
-   
+
     </details>
 
 1. Merge `release` branch back to `develop`:
-   * Open the Release PR ( PR from `release` branch targeting `develop`) which has been automatically created.
+   * Open the Release PR (PR from `release` branch targeting `develop`) which has been automatically created.
    * Resolve the conflicts (if any).
-   * In case there are changes other than updates to app and build versions after merging the changes from `develop`, the release engineer should assign as reviewers all the engineers who worked on those changes and remove the ones automatically assigned by PullAssigners. In current workflow every change integrated to release branch is supposed to be go into `develop` soon after, so differences should be minimal. Any differences might be a result of resolving conflicts or a hotfix PR not being merged previously to `develop`.
+   * In case there are changes other than updates to app and build versions after merging the changes from `develop`, the release engineer should assign as reviewers all the engineers who worked on those changes and remove the ones automatically assigned. In current workflow every change integrated to release branch is supposed to be go into `develop` soon after, so there shouldn't be any changes. Any differences might be a result of resolving conflicts, or a hotfix PR not being merged previously to `develop`.
    * Set the _Merge_ label once all the required reviewers have approved it.
 1. Update this document if any steps during the release process have changed.
 
 
-## 3. SDK Release
+## Hotfix Release
+
+### Phase 1: Initiation
+
+The hotfix is cut from the latest stable/hotfix release tag (e.g. `4.1.1` from `4.1.0`, or `4.1.2` from `4.1.1`).
+
+**⚠️ NOTE:** Only run the automated process if the previous release has been finished. In order to run the CRP, the GitHub release/tab need to exist, otherwise it will include unrelated tickets.
+
+1. Trigger manually with `/stevenson release_cutoff target:Babylon version:4.1.1 branch:{hash of last release tag}` from `#ios-build` channel
```

Comment:
Need to confirm how to reference a commit instead of a branch

Reply:
There's a way to trigger CircleCI workflows on tags (which is what we need), Stevenson may not support it yet though

Reply:
removed this for now e314053